src/actions/post_build.rs

```diff
@@ -99,13 +99,15 @@ impl PostBuildHandler {
             v.clear();
         }
 
+        let mut group = 0;
```

Comment:
style nit: could we use `for (group, msg) in messages.iter().enumerate()` instead? Seems cleaner than manually updating the `group` counter